example-playbooks/wipe_node/wipe_node.yml

```diff
@@ -92,7 +92,5 @@
       shell: |
         systemctl daemon-reload
 
-    - name: Remove Scylla packages
```

Comment:
Please, add more "why?" text to a patch description. 
AFAIU this patch is needed because there is some issue with the `package` Ansible module, isn't it?

Reply:
> Please, add more "why?" text to a patch description.

Done.

>  AFAIU this patch is needed because there is some issue with the package Ansible module, isn't it?

Yes, the `package` module doesn't allow us to remove configuration files. I've added a more detailed explanation in the patch description.